app/routes/ping.js

```diff
@@ -0,0 +1,9 @@
+const PingRoute = function(app, pkg){
+
+  app.get("/", (req, res) => {
```

Comment:
@rodrigopavezi If this is going to be the default endpoint, how will we differentiate for a file's endpoint? I thought the URL for that was going to be localhost:9494/?url=. Won't it need the same endpoint since the only difference is the `url` query parameter?